public/locales/en/app.json

```diff
@@ -85,9 +83,7 @@
   "notificationsVerifyEmail": "You need to verify your email to turn on notifications",
   "noVotes": "No one has voted for this option",
   "ok": "Ok",
-  "options": "Options",
   "participant": "Participant",
-  "participantCount_other": "{{count}} participants",
```

Comment:
So, we deal with [plurals](https://www.i18next.com/translation-function/plurals) by having these keys that are never used directly in the code but are still necessary. I believe i18n-unused can handle this ((see here)[https://github.com/mxmvshnvsk/i18n-unused/issues/13]) out of the box so not sure yet why these are being flagged 🤔

Reply:
Ah ok, then that was totally my mistake. I was searching for usages of [context](https://www.i18next.com/translation-function/context), but did not think of plurals. So I switched off the corresponding i18n-unused config just for one time. In hindsight, it should have been obvious. Will revert the last commit.